packages/immutable-arraybuffer/shim-hermes.js

```diff
@@ -28,7 +29,7 @@ if ('transfer' in arrayBufferPrototype) {
   );
 }
 
-if ('transferToImmutable' in arrayBufferPrototype) {
+if ('sliceToImmutable' in arrayBufferPrototype) {
```

Comment:
For hermes the warning would be wrong because the hermes shim would not overwrite `transferToImmutable`.

I changed both to `sliceToImmutable` because it works for both. For the main shim, `sliceToImmutable` is just as good and maintains the parallelism.